edit-post/components/header/publish-button/index.js

```diff
@@ -0,0 +1,25 @@
+import { PostPublishPanelToggle, PostPublishButton } from '@wordpress/editor';
+
+const PublishButton = function( {
```

Comment:
So this is now a button that turns into a panel? I'd sort of expect we'd have a button that toggled a panel on/off, not really one that changes from panel to button itself...